### ISSUE TITLE
label required?

### ISSUE THREAD

**Issue by valankar** [trea/caddy-loki-logger#2]
The main page has an example:

```
{
    log default {
        output loki https://myuser:password@loki.example.com {
           REGION iad  
        }
    }
}
```

However, is `label` required here? I think when I tried it without, the actual labels weren't applied and Grafana Cloud Loki complained that at least one label was needed. It could have been I misconfigured Loki, so I just wanted to check. When I added `label`, it started working.

**Comment by trea** [comment]
Seems so, then! Sorry about that! You're welcome to PR an update to the README if you'd like, otherwise I'll take another swing at it when I can.

**Comment by trea** [comment]
Was fixed by #5, thanks!
